events/src/test/java/com/redhat/runtimes/inventory/events/EventConsumerIT.java

```diff
@@ -59,6 +68,31 @@ void clear() {
     micrometerAssertionHelper.removeDynamicTimer(CONSUMED_TIMER_NAME);
   }
 
+  @Test
+  @SuppressWarnings("unchecked")
+  // This complains about not being able to find the egg channel
+  // Which is odd because it works in Prod
+  @Disabled
```

Comment:
The event TestLifeCycleManager doesn't have the egg channel added to the `InMemoryConnector` [[0]](https://github.com/RedHatInsights/insights-runtimes-inventory/blob/c80e57b585622e9da5c4cc2a7eec3ff2af9aa62f/events/src/test/java/com/redhat/runtimes/inventory/events/TestLifecycleManager.java#L34), it will need one added like the rest one does [[1]](https://github.com/RedHatInsights/insights-runtimes-inventory/blob/c80e57b585622e9da5c4cc2a7eec3ff2af9aa62f/rest/src/test/java/com/redhat/runtimes/inventory/web/TestLifecycleManager.java#L38). The test looks to pass once enabled after making this change.

[0] https://github.com/RedHatInsights/insights-runtimes-inventory/blob/c80e57b585622e9da5c4cc2a7eec3ff2af9aa62f/events/src/test/java/com/redhat/runtimes/inventory/events/TestLifecycleManager.java#L34
[1] https://github.com/RedHatInsights/insights-runtimes-inventory/blob/c80e57b585622e9da5c4cc2a7eec3ff2af9aa62f/rest/src/test/java/com/redhat/runtimes/inventory/web/TestLifecycleManager.java#L38

Reply:
Thanks for figuring that out!